### PR TITLE
Add security permissions for Kibana Notebooks

### PR DESCRIPTION
**Pull request by joshuali925**
*Issue #, if available:*

*Description of changes:*
Add Kibana Notebooks permissions to `CLUSTER_PERMISSIONS`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/configuration/constants.tsx [hunk 1]
```diff
@@ -61,6 +61,11 @@ export const CLUSTER_PERMISSIONS: string[] = [
   'cluster:admin/opendistro/alerting/monitor/get',
   'cluster:admin/opendistro/alerting/monitor/search',
   'cluster:admin/opendistro/alerting/monitor/write',
+  'cluster:admin/opendistro/notebooks/create',
+  'cluster:admin/opendistro/notebooks/update',
+  'cluster:admin/opendistro/notebooks/delete',
```
Comment (sandeshkr419) [inline public/apps/configuration/constants.tsx:66]: Arrangement of all the added permissions should be in alphabetical order to maintain consistency.

**Review by sandeshkr419 — CHANGES_REQUESTED** [review]
Ordering of added permissions.

**Comment by davidlago** [comment]
@joshuali925 apologies for the delay here. Could you please provide more details as to why this change is needed, and if it is related to a change/feature in a different repository a link to it as well?

If this is still needed (it was created over a year ago, sorry!) please rebase from `main` and resubmit.

**Comment by joshuali925** [comment]
> @joshuali925 apologies for the delay here. Could you please provide more details as to why this change is needed, and if it is related to a change/feature in a different repository a link to it as well?
> 
> If this is still needed (it was created over a year ago, sorry!) please rebase from `main` and resubmit.

@davidlago no worries. I thought this was merged long time ago, but we didn't see any issue in notebooks permissions i guess because [PR to backend](https://github.com/opensearch-project/security/pull/903) was merged. 

This PR was to add permissions for [notebooks plugin](https://github.com/opensearch-project/dashboards-notebooks) and is no longer relevant. The notebooks plugin is deprecated to the observability plugin (security PR #851). Closing this, thanks.

**Pull request closed without merging**